index.html

```diff
@@ -61,6 +61,164 @@ <h2 class="lets-connect">LET’S CONNECT</h2>
+          Stories </h2>
+        <ul class="menu2">
+          <li><a class="canopy" href="#">CANOPY</a> </li>
+          <li><img src="./images/Counter.svg" alt="counter"></li>
+          <li><a class="back-end" href="#">Back End Dev</a></li>
+          <li><img src="./images/Counter.svg" alt="counter"></li>
+          <li> <a class="year" href="#">Year</a></li>
+        </ul>
+        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups
+          required.</p>
+        <ul class="html-tags">
+          <li><a href=""><img src="./images/html.svg" alt="html-tag"></a></li>
+          <li><a href=""><img src="./images/css.svg" alt="css-tag"></a></li>
+          <li><a href=""><img src="./images/javascript.svg" alt="javascript-tag"></a></li>
+        </ul>
+        <button type="button" class="btn-card">See Project</button>
+      </section>
+    </section>
+    
+    <Section class="about-myself">
+      <div class="header-myself">
+          <h2 class="heading-about">About <br> Myself</h2>
+      </div>
+
+      <div class="paragraph3">
+          <p class="p3">Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If you like what you see and
+              have a
+              project you need coded, don’t hestiate to contact me.
+          </p>
+          <h2 class="lets-connect">LET’S CONNECT</h2>
+      </div>
+
+      <div class="social-footer">
+          <ul class="social-media">
+              <li><a href="#"><img src="./images/twitter.png" alt="twitter-icon"></a></li>
+              <li><a href="#"><img src="./images/linkedin.png" alt="linkedin-icon"></a></li>
+              <li><a href="#"><img src="./images/mail.png" alt="mail-icon"></a></li>
+              <li><a href="#"><img src="./images/github.png" alt="github-icon"></a>
+              <li>
+              <li><a href="#"><img src="./images/fingers.png" alt="fingers-icon"></a>
+              <li>
+          </ul>
+      </div>
+      <button type="button" class="button1">Get my resume</button>
+
+      <div class="languages">
+          <h2>Languages</h2>
+          <a href="#" aria-label="down-arrow" class="arrow-down"><img src="./images/arrow-down.svg" alt=""></a>
+      </div>
+      <div class="language-list">
+          <div class="java1">
+              <a href="#" class="java-script"> <img src="./images/java-script.svg" alt="java-script"></a>
+              <p class="java-tex">JavaScript</p>
+          </div>
+          <div class="java1">
+              <a href="#" class="java-script"> <img src="./images/HTML-icon.svg" alt="java-script"></a>
+              <p class="java-tex">HTML</p>
+          </div>
+          <div class="java1">
+              <a href="#" class="java-script"> <img src="./images/CSS-icon.svg" alt="java-script"></a>
+              <p class="java-tex">CSS</p>
+          </div>
+      </div>
```

Comment:
- [ ] Instead of using `div` tags for language list please use `ul` and `li` tags due to some SEO purpose :+1: